.gitlab-ci.yml

```diff
@@ -22,6 +22,36 @@ deploy frankfurt:
     - master
     - frankfurt
   script:
-    - kubectl_1.13 apply -f frankfurt/nucleus-dev
-    - kubectl_1.13 rollout status -f frankfurt/nucleus-dev/deploy.yaml -w --timeout=10m
-    - kubectl_1.13 apply -f frankfurt/nucleus-stage
+    - kubectl_1.11 apply -f frankfurt/nucleus-dev
+    - kubectl_1.11 rollout status -f frankfurt/nucleus-dev/deploy.yaml -w --timeout=10m
+
+deploy frankfurt stage:
+  variables:
+    KUBECONFIG: /home/gitlab-runner/.kube/frankfurt.kubeconfig
+  tags:
+    - aws
+    - frankfurt
+    - meao
+  only:
+    - master
+    - frankfurt
+    - stage
+  script:
+    - kubectl_1.11 apply -f frankfurt/nucleus-stage
+    - kubectl_1.11 rollout status -f frankfurt/nucleus-stage/deploy.yaml -w --timeout=10m
+
+deploy frankfurt prod:
+  variables:
+    KUBECONFIG: /home/gitlab-runner/.kube/frankfurt.kubeconfig
+  tags:
+    - aws
+    - frankfurt
+    - meao
+  only:
+    - master
```

Comment:
Same here - do we want `master` updating prod?

Reply:
yes, but kubectl apply here should be idempotent if there are no changes in `frankfurt/nucleus-prod`

Reply:
Ah, right, right. The config files themselves will determine which apps are actually changed.